=== tests/cases/compiler/flowAfterFinally1.ts ===
declare function openFile(): void
>openFile : () => 无值

declare function closeFile(): void
>closeFile : () => 无值

declare function someOperation(): {}
>someOperation : () => {}

var result: {}
>result : {}

openFile()
>openFile() : 无值
>类型别名 :=> 无值
>openFile : () => 无值

try {
  result = someOperation()
>result = someOperation() : {}
>result : {}
>someOperation() : {}
>someOperation : () => {}

} finally {
  closeFile()
>closeFile() : 无值
>类型别名 :=> 无值
>closeFile : () => 无值
}

result // should not error here
>result : {}

